### ISSUE TITLE
Multi-line settings aren't ignored properly by sync

### ISSUE THREAD

**Issue by connorshea** [shanalikhan/code-settings-sync#701]
🐛 **Describe the bug**  
When using the new per-platform sync feature, settings that run for more than one line aren't commented-out properly.

🌴 **Visual Studio Code Version :** Code 1.28.2 (7f3ce96ff4729c91352ae6def877e59c561f4850)
🌴 **Code Settings Sync Version :** 3.2.0
🌴 **Standard or Insiders :** Standard
🌴 **OSS or Portable :** neither?
🌴 **Operating System :** macOS Mojave 10.14.0
🌴 **Occurs On:** Download
🌴 **Proxy Enabled:** No

📰 **To Reproduce**

1. Have VS Code with sync set up on two different machines (can be the same OS, just use the `host` option instead of `os`), we'll call them Machine A and Machine B.
2. Create a user settings file on Machine A (in my case, Mac) like so:
```jsonc
{
  // @sync os=mac
  "editor.quickSuggestions": {
    "other": true,
    "comments": true,
    "strings": true
  }
}
```
3. Sync the settings file to Machine B (Windows for me)
4. This is what you'll end up with on Machine B:
```jsonc
{
  // @sync os=mac
  // "editor.quickSuggestions": {
    "other": true,
    "comments": true,
    "strings": true
  }
}
```

💪 **Expected behavior**
The setting should be properly commented-out on Machine B.

```jsonc
{
  // @sync os=mac
  // "editor.quickSuggestions": {
  //  "other": true,
  //  "comments": true,
  //  "strings": true
  // }
}
```

**Comment by shanalikhan** [comment]
@ioprotium 
One more to go :smile:

**Comment by ioprotium** [comment]
I will add that tomorrow! PR to 3.2.1 right?

**Comment by shanalikhan** [comment]
Yes, currently settings sync stands at v3.2.0

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by shanalikhan** [comment]
@ioprotium any update on this? :)

**Comment by ioprotium** [comment]
@shanalikhan Sorry for the delay. Yes I changed the code that parses the settings, I didn't contemplate the case for multi line settings, my bad. I will test on my environment and then drop the PR

**Comment by shanalikhan** [comment]
Fixed in v3.2.5
@connorshea

**Comment by connorshea** [comment]
Thanks!
